androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/GetAccessTokenTask.java

```diff
@@ -50,7 +50,9 @@ protected String requestUrl() {
     protected void buildParams() {
         paramsMap.put("grant_type", "client_credentials");
         paramsMap.put("client_id", clientId);
-        paramsMap.put("client_secret", clientSecret);
+        if (clientSecret != null) {
```

Comment:
I don't think we should even use `client_secret` for best security practices. @diegoserranoa what do you think?

Reply:
Umm yes, but for current use, we are trying to have both. But while using `client_secret` user will get `depreciated` warning. Ok let's see what @diegoserranoa says

Reply:
@prabhatjha @ambarwootric I was trying to go for the approach of a slow transition so users can still use this method. If we want user to completely use the new method, then you should probably just go to adding the new method @ambarwootric.

Reply:
@diegoserranoa ok, let's go with your approach. I guess it's ready to ship then.